test.py

```diff
@@ -1015,12 +1015,15 @@ class ToolTest(Test):
 
     def __init__(self, test_no: int, shortname: str, suite) -> None:
         super().__init__(test_no, shortname, suite)
-        self.path = "pytest"
+        launcher = self.suite.cfg.get("launcher", "pytest")
```

Comment:
I don't think we need this option. We can just unconditionally launch all tool test suites in a new namespace and mention this in the suite docs above.

Reply:
the reason why it designed this was is to decouple the nodetool from the unshare, so we can just launch it using `pytest`.